tests/conftest.py

```diff
@@ -235,26 +234,35 @@ def bed_file():
         "status": "archived",
     }
 
+
 @pytest.fixture
 def s3_uri():
     return "s3://encode-public/2017/07/02/a55e8a04-d40d-4715-8bee-7b23dc4f1050/ENCFF603JQT.bed.gz"
 
+
 @pytest.fixture
-def reader_chip_seq():    
-    file = gzip.open("./tests/data/ENCFF141SAU.bed.gz", mode='rt')
-    file_reader = csv.reader(file, delimiter='\t')
+def reader_chip_seq():
+    file = gzip.open("./tests/data/ENCFF141SAU.bed.gz", mode="rt")
+    file_reader = csv.reader(file, delimiter="\t")
     return file_reader
 
+
 @pytest.fixture
-def reader_eqtls_grch38():   
-    file = gzip.open("./tests/data/ENCFF776SWB_eqtls_grch38.bed.gz", mode='rt')
-    file_reader = csv.reader(file, delimiter='\t')
+def reader_eqtls_grch38():
+    file = gzip.open("./tests/data/ENCFF776SWB_eqtls_grch38.bed.gz", mode="rt")
+    file_reader = csv.reader(file, delimiter="\t")
     return file_reader
 
+
 @pytest.fixture
-def reader_snp():    
-    file = gzip.open("./tests/data/test_snp_file.bed.gz", mode='rt')
-    file_reader = csv.reader(file, delimiter='\t')
+def reader_snp():
+    file = gzip.open("./tests/data/test_snp_file.bed.gz", mode="rt")
+    file_reader = csv.reader(file, delimiter="\t")
+    return file_reader
+
+@pytest.fixture
+def reader_footprint_grch38():
+    file_reader = csv.reader(open('./tests/data/footprint_grch38.bed'), delimiter="\t")
```

Comment:
We should decide on single or double quotes (I have no opinion which) and stick to it. Just do not mix.